plugins/covid/lab.py

```diff
@@ -115,16 +175,24 @@ def get_result(test):
     return observations[0].observation_value
 
 
-def get_resulted_date(test):
+def get_resulted_datetime(test):
     """
     Given the plugins.labtests.models.LabTest instance TEST, which is a
-    COVID 19 test, return the date of the relevant observation
+    COVID 19 test, return the datetime of the relevant observation
     """
     covid_test = _get_covid_test(test)
     observations = test.observation_set.filter(
         observation_name=covid_test.OBSERVATION_NAME
     )
-    return observations[0].observation_datetime.date()
+    return observations[0].observation_datetime
```

Comment:
Do all observations definitely have a datetime? Logically they should but we can't guarantee it.

Reply:
0.07% of tests have a null datetime. 
100% of these also have a null observation_name